cli/flags.rs

```diff
@@ -509,6 +515,23 @@ fn resolve_paths(paths: Vec<String>) -> Vec<String> {
   out
 }
 
+fn resolve_urls(urls: Vec<String>) -> Vec<String> {
+  let mut out: Vec<String> = vec![];
+  for urlstr in urls.iter() {
+    let result = Url::from_str(urlstr);
+    if result.is_err() {
+      eprintln!("Bad Url: {}", urlstr);
+      continue;
+    }
+    let mut full_url = String::from(result.unwrap().as_str());
+    if full_url.len() > 1 && full_url.ends_with('/') {
```

Comment:
Why is this `pop` needed? Can you add a comment?

Reply:
for prefix matching, it's the same thing as in files whitelist
it removes everything after '/' from the address, so e.g /foo/bar -> /foo